delocate/tools.py

```diff
@@ -135,6 +135,10 @@ def _line0_says_object(line0, filename):
     if line0.startswith('Archive :'):
         # nothing to do for static libs
         return False
+    if line0.endswith("'%s': The file was not recognized as a valid object file." % filename):
```

Comment:
Can you find a way of making tests for these guys?  Small object files you can commit to test against, for example?